Szwalnia/PolkiNaRegaleForm.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Data;
+using System.Drawing;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Forms;
+
+namespace Szwalnia
+{
+    public partial class PolkiNaRegaleForm : Form
+    {
+        public SzwalniaEntities szwalnia_passed;
+        public PolkiNaRegaleForm(SzwalniaEntities szwalnia_passed, string oznaczenie)
+        {
+            InitializeComponent();
+            vPolki_na_regalach storage = szwalnia_passed.vPolki_na_regalach.Where(regal => regal.Oznaczenie == oznaczenie).First();
+
+            lblTitle.Text = "REGAL " + storage.Oznaczenie;
+
+            dgvRegaly.DataSource = szwalnia_passed.vPolki_na_regalach.Where(regal => regal.Oznaczenie == storage.Oznaczenie).ToList();
+        }
+        private void dgvRegaly_CellContentDoubleClick(object sender, DataGridViewCellEventArgs e)
+        {
+            if (dgvRegaly.Rows[e.RowIndex].Cells[e.ColumnIndex].Value != null)
```

Comment:
czy to sprawdzenie jest zamierzone? jeśli dowolna komórka będzie nullem i zostanie kliknięta to nic się nie wydarzy

Reply:
Cóż - nieudolnie próbowałam sobie poradzić z przejściem do nowego formularza. Przerzucę się na ten przedstawiony przez Pana sposób i nie będę drążyć tamtego

Reply:
To nie jest nieudolne- być może kiedyś takie działanie będzie pożądane.
Natomiast należy dokładnie rozumieć pisany przez siebie kod i starać znaleźć wszystkie konsekwencje jego działania